controller/userController.js

```diff
@@ -0,0 +1,101 @@
+const {todos} = require("../models");
+
+class Controller {
+
+    static async AddGettodos (req, res){
```

Comment:
sebenernya semua requirement udah cakep nih eva!  mantap, tapi ada beberapa best practice yg masih bisa diterapin contohnya disini, mungkin kamu penamaan filesnya ngikutin lecture ya, jadi userController, tp user itu prefix aja wkwk jadi sebenernya disesuaikan sama context didalemnya, kaya ini better jadi todoController.js, biar nanti kalau aplikasi kamu udah super besar pas ada improvement di todo tinggal arahin ke controller ini
termasuk nama static functionya, sebelumnya AddUser itu karena tujuan function tersebut untuk add user kedalam db, 
nah ini bisa diimplemet juga di controller ini, misalnya kalau buat get yaudah bikinya GetTodos, kalau ada spesifik bisa GetTodoById, kalau nambahin todos bisa, AddTodo. jadi dari nama function aja kamu udah tau si static ini itu tugasnya apa.

tapi sofar semua requirement udah terpenuhi koo, keep it up buat tugas berikutnya!!